### DISCUSSION TITLE
https://mokkapps.de/vue-tips/detecting-server-vs-client-side-code-execution-in-nuxt

### DISCUSSION THREAD

**Discussion by giscus** [Mokkapps/mokkapps-de-discussions#6] [Announcements]
# https://mokkapps.de/vue-tips/detecting-server-vs-client-side-code-execution-in-nuxt

Nuxt 3 provides a way to detect whether your code is running on the server or client-side.

https://mokkapps.de/vue-tips/detecting-server-vs-client-side-code-execution-in-nuxt

**Comment by cn-2k** [comment]
Little update: now ``process*`` is ``import.meta*``.

Refer to https://github.com/nuxt/nuxt/pull/26611

**Reply by cn-2k** [reply]
nice post btw :)

**Reply by Mokkapps** [reply]
Thanks 👍🏻
